### PR TITLE
Feature/10509 session storage bug

### PR DESCRIPTION
**Pull request by samMcCarthyBJSS**
Story [AB#10509](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/10509)
Task [AB#10624](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/10624)

### CODE REVIEW

File: app/pages/sections/order-items/catalogue-solutions/routes.js [hunk 1]
```diff
@@ -113,6 +113,17 @@ export const catalogueSolutionsRoutes = (authProvider, addContext, sessionManage
...
+        sessionManager.saveToSession({ req, key: sessionKeys.selectedItemId, value: undefined });
+        sessionManager.saveToSession({ req, key: sessionKeys.selectedItemName, value: undefined });
+        sessionManager.saveToSession({ req, key: sessionKeys.solutionPrices, value: undefined });
+        sessionManager.saveToSession({ req, key: sessionKeys.selectedPriceId, value: undefined });
+        sessionManager.saveToSession(
+          { req, key: sessionKeys.selectedRecipients, value: undefined },
+        );
+        sessionManager.saveToSession(
+          { req, key: sessionKeys.plannedDeliveryDate, value: undefined },
+        );
```
Comment (bjss-chris) [inline app/pages/sections/order-items/catalogue-solutions/routes.js:126]: Would `sessionManager.clearSession` be of any use here or are there some details that need to remain in session at this point (i.e. it's too destructive)?

**Review by bjss-chris — COMMENTED** [review]
Just one question. Otherwise, looks good.

File: app/pages/sections/order-items/catalogue-solutions/routes.js (hunk 1 above)
Comment (samMcCarthyBJSS) [reply app/pages/sections/order-items/catalogue-solutions/routes.js:126]: I did think about that but reckon we'll need them for other bits so probs better to not get rid of everything

Comment (bjss-chris) [reply app/pages/sections/order-items/catalogue-solutions/routes.js:126]: Do you think it's worth extracting this to a separate helper then? Could it be useful for additional and/or associated services?

Comment (samMcCarthyBJSS) [reply app/pages/sections/order-items/catalogue-solutions/routes.js:126]: Hm if it does get reused we should remove it but it's only used here currently
they won't have the solutions session key I don't think either

**Review by bjss-chris — APPROVED** [review]

**Pull request merged**